network/substreams.yaml

```diff
@@ -1,68 +1,97 @@
 specVersion: v0.1.0
 package:
-  name: "messari_networks_substreams"
+  name: messari_networks_substreams
   version: v0.1.0
 
 imports:
-  eth: https://github.com/streamingfast/sf-ethereum/releases/download/v0.10.2/ethereum-v0.10.4.spkg
-
-protobuf:
-  files:
-    - network.proto
-  importPaths:
-    - ./proto/v1
+  eth: "https://github.com/streamingfast/sf-ethereum/releases/download/v0.10.2/ethereum-v0.10.4.spkg"
 
 binaries:
   default:
     type: wasm/rust-v1
-    file: ../target/wasm32-unknown-unknown/release/messari_networks_substreams.wasm
+    file: "../target/wasm32-unknown-unknown/release/network.wasm"
 
 modules:
-  - name: store_cumulative_values
+  - name: unique_tracking
+    kind: store
+    initialBlock: 1
```

Comment:
initial block 0?

Reply:
Block 0 is faulty. Not sure what is going on with it but it doesn't even have a timestamp associated with the block so I thought I'd just leave it out. (All other blocks seem to work fine so far)